StartupShutdownServiceTest testHistoryTablesMatch fails

<!--
Please read the Contributing guidelines (https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/docs/CONTRIBUTING.md) before submitting an issue. You don't have to remove this comment or any other comment from this issue as they will automatically be hidden.
-->
### Problem Description
<!-- Mandatory -->

This tests fails in Github Actions and on Jenkins (therefore blocking CI pipeline).

```
Fehlschlag: de.symeda.sormas.backend.common.StartupShutdownServiceTest.testHistoryTablesMatch

Fehlermeldung: Container startup failed

Stacktrace

org.testcontainers.containers.ContainerLaunchException: Container startup failed
	at de.symeda.sormas.backend.common.StartupShutdownServiceTest.testHistoryTablesMatch(StartupShutdownServiceTest.java:121)
Caused by: org.testcontainers.containers.ContainerFetchException: Can't get Docker image: RemoteDockerImage(imageName=<resolving>, imagePullPolicy=DefaultPullPolicy(), imageNameSubstitutor=org.testcontainers.utility.ImageNameSubstitutor$LogWrappedImageNameSubstitutor@4ad0d8d4)
	at de.symeda.sormas.backend.common.StartupShutdownServiceTest.testHistoryTablesMatch(StartupShutdownServiceTest.java:121)
Caused by: com.github.dockerjava.api.exception.DockerClientException: Could not build image: The command '/bin/sh -c apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.13/main/ musl=1.2.2-r1 musl-dev=1.2.2-r1 &&     apk add --no-cache openssl=1.1.1o-r0 curl=7.79.1-r1 tzdata=2021a-r0 py-pip=20.3.4-r1 python3-dev=3.9.5-r2      postgresql-dev=13.6-r0 postgresql-contrib=13.6-r0 make=4.3-r0 gcc=10.3.1_git20210424-r2 py3-psutil=5.8.0-r1  &&     pip install --no-cache-dir pgxnclient==1.3.2 &&     pgxnclient install temporal_tables' returned a non-zero code: 2
	at de.symeda.sormas.backend.common.StartupShutdownServiceTest.testHistoryTablesMatch(StartupShutdownServiceTest.java:121)

Standard Ausgabe (STDOUT)

2022-06-08 12:15:48,110 ERROR (org.testcontainers.images.builder.ImageFromDockerfile$1.java:100) - The command '/bin/sh -c apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.13/main/ musl=1.2.2-r1 musl-dev=1.2.2-r1 &&     apk add --no-cache openssl=1.1.1o-r0 curl=7.79.1-r1 tzdata=2021a-r0 py-pip=20.3.4-r1 python3-dev=3.9.5-r2      postgresql-dev=13.6-r0 postgresql-contrib=13.6-r0 make=4.3-r0 gcc=10.3.1_git20210424-r2 py3-psutil=5.8.0-r1  &&     pip install --no-cache-dir pgxnclient==1.3.2 &&     pgxnclient install temporal_tables' returned a non-zero code: 2
```

### Proposed Solution
<!-- Mandatory -->

### Possible Alternatives
<!-- Optional -->

### Additional Information
<!-- Optional -->

```

The problem is still not fixed on Jenkins:
```
org.testcontainers.containers.ContainerLaunchException: Container startup failed
	at de.symeda.sormas.backend.common.StartupShutdownServiceTest.testHistoryTablesMatch(StartupShutdownServiceTest.java:134)
Caused by: org.testcontainers.containers.ContainerFetchException: Can't get Docker image: RemoteDockerImage(imageName=<resolving>, imagePullPolicy=DefaultPullPolicy(), imageNameSubstitutor=org.testcontainers.utility.ImageNameSubstitutor$LogWrappedImageNameSubstitutor@68c07805)
	at de.symeda.sormas.backend.common.StartupShutdownServiceTest.testHistoryTablesMatch(StartupShutdownServiceTest.java:134)
Caused by: com.github.dockerjava.api.exception.DockerClientException: Could not build image: The command '/bin/sh -c apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.13/main/ musl=1.2.2-r1 musl-dev=1.2.2-r1 &&     apk add --no-cache openssl=1.1.1o-r0 curl=7.79.1-r1 tzdata=2022a-r0 py-pip=20.3.4-r1 python3-dev=3.9.5-r2      postgresql-dev=13.6-r1 postgresql-contrib=13.6-r1 make=4.3-r0 gcc=10.3.1_git20210424-r2 py3-psutil=5.8.0-r1  &&     pip install --no-cache-dir pgxnclient==1.3.2 &&     pgxnclient install temporal_tables' returned a non-zero code: 2
	at de.symeda.sormas.backend.common.StartupShutdownServiceTest.testHistoryTablesMatch(StartupShutdownServiceTest.java:134)
```

I again deactivated that test to hopefully unblock the CI pipeline.

--- a/sormas-backend/src/test/java/de/symeda/sormas/backend/common/StartupShutdownServiceTest.java
+++ b/sormas-backend/src/test/java/de/symeda/sormas/backend/common/StartupShutdownServiceTest.java
@@ -127,8 +127,6 @@ public class StartupShutdownServiceTest extends BaseBeanTest {
 	 * @throws URISyntaxException
 	 *             if the path to a resource is not syntactically correct
 	 */
-	// TODO #9446: Fix failing test
-	@Ignore
 	@Test
 	public void testHistoryTablesMatch() throws IOException, URISyntaxException {
 

--- a/sormas-backend/src/test/resources/testcontainers/Dockerfile
+++ b/sormas-backend/src/test/resources/testcontainers/Dockerfile
@@ -1,5 +1,6 @@
 FROM postgres:10.18-alpine
 
+# hadolint ignore=DL3018,DL3013
 RUN apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.13/main/ musl musl-dev && \
     apk add --no-cache openssl curl tzdata py-pip python3-dev  \
     postgresql-dev postgresql-contrib make gcc py3-psutil  && \

--- a/sormas-backend/src/test/resources/testcontainers/Dockerfile
+++ b/sormas-backend/src/test/resources/testcontainers/Dockerfile
@@ -1,7 +1,7 @@
 FROM postgres:10.18-alpine
 
 # hadolint ignore=DL3018,DL3013
-RUN apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.13/main/ musl musl-dev && \
+RUN apk add --no-cache --repository  http://dl-cdn.alpinelinux.org/alpine/v3.13/main/ 'musl<1.2.2-r3' 'musl-dev<1.2.2-r3' && \
     apk add --no-cache openssl curl tzdata py-pip python3-dev  \
     postgresql-dev postgresql-contrib make gcc py3-psutil  && \
     pip install --no-cache-dir pgxnclient && \

--- a/sormas-backend/src/test/resources/testcontainers/Dockerfile
+++ b/sormas-backend/src/test/resources/testcontainers/Dockerfile
@@ -1,9 +1,9 @@
 FROM postgres:10.18-alpine
 
-RUN apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.13/main/ musl=1.2.2-r1 musl-dev=1.2.2-r1 && \
-    apk add --no-cache openssl=1.1.1o-r0 curl=7.79.1-r1 tzdata=2022a-r0 py-pip=20.3.4-r1 python3-dev=3.9.5-r2  \
-    postgresql-dev=13.6-r1 postgresql-contrib=13.6-r1 make=4.3-r0 gcc=10.3.1_git20210424-r2 py3-psutil=5.8.0-r1  && \
-    pip install --no-cache-dir pgxnclient==1.3.2 && \
+RUN apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.13/main/ musl musl-dev && \
+    apk add --no-cache openssl curl tzdata py-pip python3-dev  \
+    postgresql-dev postgresql-contrib make gcc py3-psutil  && \
+    pip install --no-cache-dir pgxnclient && \
     pgxnclient install temporal_tables
 
 COPY setup_sormas_db.sh /docker-entrypoint-initdb.d/

